diff --git a/config/storybook/main.js b/config/storybook/main.js
index bc381eee10cc122e74d999b616784f228388d53a..45747dc6dbef05d24faa01cc9c9ad08b3658f9d9 100644
--- a/config/storybook/main.js
+++ b/config/storybook/main.js
@@ -57,14 +57,8 @@ const baseConfig = {
 
     return sbConfig
   },
-  webpackFinal: (sbConfig, { configType }) => {
-    // configType is 'PRODUCTION' or 'DEVELOPMENT', why shout?
-    const isEnvProduction =
-      configType && configType.toLowerCase() === 'production'
-
-    const rwConfig = isEnvProduction
-      ? require('@redwoodjs/core/config/webpack.production')
-      : require('@redwoodjs/core/config/webpack.development')
+  webpackFinal: (sbConfig) => {
+    const rwConfig = require('@redwoodjs/core/config/webpack.production')
 
     // We replace imports to "@redwoodjs/router" with our own implementation in "@redwoodjs/testing"
     sbConfig.resolve.alias['@redwoodjs/router$'] = require.resolve(
@@ -109,7 +103,7 @@ const baseConfig = {
     // ** PLUGINS **
     sbConfig.plugins = [
       ...sbConfig.plugins,
-      ...getSharedPlugins(isEnvProduction),
+      ...getSharedPlugins(true),
     ]
 
     // ** LOADERS **
