diff --git a/config/storybook/main.js b/config/storybook/main.js
index bc381eee10cc122e74d999b616784f228388d53a..26191b6e5d70bd5ae14822e61794d61874329534 100644
--- a/config/storybook/main.js
+++ b/config/storybook/main.js
@@ -1,3 +1,4 @@
+//@ts-check
 const fs = require('fs')
 const path = require('path')
 
@@ -14,7 +15,6 @@ const config = getConfig()
 
 const rwjsPaths = getPaths()
 
-const staticAssetsFolder = path.join(getPaths().web.base, 'public')
 
 function isPackageInstalled(alias) {
   try {
@@ -57,10 +57,9 @@ const baseConfig = {
 
     return sbConfig
   },
-  webpackFinal: (sbConfig, { configType }) => {
-    // configType is 'PRODUCTION' or 'DEVELOPMENT', why shout?
-    const isEnvProduction =
-      configType && configType.toLowerCase() === 'production'
+  webpackFinal: (sbConfig) => {
+    // hardcoding `isEnvProduction = true` hack to get building on netlify to work
+    const isEnvProduction = true
 
     const rwConfig = isEnvProduction
       ? require('@redwoodjs/core/config/webpack.production')
@@ -131,11 +130,7 @@ const baseConfig = {
     sbConfig = withEmotionVersionFallback(sbConfig)
 
     return sbConfig
-  },
-  // only set staticDirs when running Storybook process; will fail if set for SB --build
-  ...(process.env.NODE_ENV !== 'production' && {
-    staticDirs: [`${staticAssetsFolder}`],
-  }),
+  }
 }
 
 const mergeUserStorybookConfig = (baseConfig) => {
