diff --git a/dist/cjs/autoTitle.js b/dist/cjs/autoTitle.js
index d259344d272b278186b7e9ba4deeb94a3dab8d49..9ee6bb28633ca1fb2dbaa72b925d0b9ed69c7d8e 100644
--- a/dist/cjs/autoTitle.js
+++ b/dist/cjs/autoTitle.js
@@ -117,7 +117,8 @@ var userOrAutoTitleFromSpecifier = function userOrAutoTitleFromSpecifier(fileNam
   // slash makes sure we always handle paths with unix-style forward slash


-  var normalizedFileName = (0, _slash.default)(fileName);
+  console.error("cjs")
+  var normalizedFileName = (0, _slash.default)(String(fileName));

   if (importPathMatcher.exec(normalizedFileName)) {
     if (!userTitle) {
diff --git a/dist/esm/autoTitle.js b/dist/esm/autoTitle.js
index b906a20ef9ccc81dd2cce84a25e5f03c4bfc7fff..f793c6cf15e30e796a1588f1200272b05d09c376 100644
--- a/dist/esm/autoTitle.js
+++ b/dist/esm/autoTitle.js
@@ -92,8 +92,8 @@ export var userOrAutoTitleFromSpecifier = function userOrAutoTitleFromSpecifier(
       titlePrefix = _ref$titlePrefix === void 0 ? '' : _ref$titlePrefix; // On Windows, backslashes are used in paths, which can cause problems here
   // slash makes sure we always handle paths with unix-style forward slash

-
-  var normalizedFileName = slash(fileName);
+  console.error("esm")
+  var normalizedFileName = slash(String(fileName));

   if (importPathMatcher.exec(normalizedFileName)) {
     if (!userTitle) {
diff --git a/dist/modern/autoTitle.js b/dist/modern/autoTitle.js
index 1f1f3bb46e2898171c3cf2930addab9d8f99a341..2f4f79714b1b4a210c705383b30b98e73ed79710 100644
--- a/dist/modern/autoTitle.js
+++ b/dist/modern/autoTitle.js
@@ -51,7 +51,8 @@ export const userOrAutoTitleFromSpecifier = (fileName, entry, userTitle) => {
   } = entry || {}; // On Windows, backslashes are used in paths, which can cause problems here
   // slash makes sure we always handle paths with unix-style forward slash

-  const normalizedFileName = slash(fileName);
+  console.error("modern")
+  const normalizedFileName = slash(String(fileName));

   if (importPathMatcher.exec(normalizedFileName)) {
     if (!userTitle) {
